platform/src/main/java/org/hillview/maps/FindDeltaTableFilesMap.java

```diff
@@ -0,0 +1,41 @@
+package org.hillview.maps;
+
+import org.hillview.dataset.api.IMap;
+import org.hillview.storage.FileSetDescription;
+import org.hillview.storage.IFileReference;
+import org.hillview.utils.HillviewLogger;
+import org.hillview.utils.Utilities;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FindDeltaTableFilesMap<T> implements IMap<T, List<IFileReference>> {
```

Comment:
Let's rename this class into ExtractWorkerFilesMap. It does not really find anything.

Reply:
Done